src/frontend/src/flows/recovery/displaySeedPhrase.ts

```diff
@@ -102,6 +103,7 @@ const init = (): Promise<void> =>
           seedCopy.classList.add("is-copied");
         })
         .catch((e) => {
+          toast.error("Unable to copy seed phrase, check console for details");
```

Comment:
```suggestion
          toast.error("Unable to copy seed phrase");
```

I think anyone with some dev knowledge will know to check the console, but for everyone else this will just be confusing